packages/stryker-api/src/core/StrykerOptions.ts

```diff
@@ -37,6 +37,11 @@ interface StrykerOptions {
   configFile?: string;
 
   /**
+   * Options to pass to the reporter
+   */
+  reporterOptions?: Object;
```

Comment:
This is not needed. The idea is that the typed `StrykerOptions` properties are only used for the main stryker options. Any options the reporters use need to go in their own seperate objects. For example `{ clearTextReporter: { realtime: true } }`. On line 4 of the StrykerOptions, you can see that any additional config is allowed: ` [customConfig: string]: any;`. This makes the stryker config essentially a dictionary of config.

The html reporter is doing the same thing. See the [readme](https://github.com/stryker-mutator/stryker/tree/master/packages/stryker-html-reporter#full-config-example) and the [`HtmlReporter.ts`](https://github.com/stryker-mutator/stryker/blob/master/packages/stryker-html-reporter/src/HtmlReporter.ts#L66) for more info. 

Leaving it out of the config will make it more easier for us to spin it of in a separate package later if we want.